src/js/components/Anchor/__tests__/Anchor-test.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import 'jest-styled-components';
+
+import { Grommet } from '../../Grommet';
+import { Anchor } from '../';
+
+import { findAllByType } from '../../utils';
+
+test('Anchor renders', () => {
+  const component = renderer.create(
+    <Grommet>
+      <Anchor href='#' label='Test' />
```

Comment:
Can we have an `<Anchor />` with no properties? To make sure our defaults are right.